app/views/places/_form.html.erb

```diff
@@ -1,27 +1,29 @@
-<%= form_with(model: place, local: true) do |form| %>
-  <% if place.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(place.errors.count, "error") %> prohibited this place from being saved:</h2>
+<div class="container">
+  <%= form_with(model: place, local: true) do |form| %>
+    <% if place.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(place.errors.count, "error") %> prohibited this place from being saved:</h2>
```

Comment:
translation is missing or you can remove this test and only show error messages.

Reply:
Still missing translation